mautic/mautic
# [UI/UX] Self-hosted font

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [X]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #13139

#### Description:

Changing the font to improve compatibility and self-hosting it would involve selecting a more universally compatible typeface and hosting it on the platform's servers to optimize font rendering and loading speed.

We are chaning from Open Sans to Source Sans Pro (already used on marketing resources).

Implementing this change would likely lead to improved readability, cross-device compatibility, and faster page load times, ultimately enhancing the overall user experience and ensuring we do not depend on Google Fonts servers.

Compliance: The GDPR sets strict rules on how personal data should be handled. By hosting fonts locally, we can also ensure that Mautic is closer to a compliance with these rules. For example, the GDPR requires companies to have a valid justification for collecting and processing personal data. By hosting fonts locally, we can avoid the need to collect unnecessary personal data.

When end users visit a website that incorporates Google Fonts, their browsers send HTTP requests to the Google Fonts API. The HTTP requests include the IP address used by the respective user to access the Internet, the URL requested on the Google server, and the HTTP headers, including the user agent that describes the version of the Internet browser and the operating system of the site visitors, as well as the referrer (that is, the web page on which the Google font should be displayed).


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. You may notice a small difference in the font being used in addition to a considerable improvement in its readability in smaller texts (such as the side menu).
